integration_tests/conftest.py

```diff
@@ -62,3 +62,6 @@ def pytest_addoption(parser):
     parser.addoption(
         "--pyarrow_test", action='store_true', default=False, help="if enable pyarrow tests"
     )
+    parser.addoption(
```

Comment:
Seems not needed.
The following will work:

in integration_tests/pytest.ini
```
+    test_in_premerge_CI_with_non_UTC_TZ: Mark cases will run in the premerge CI with non-UTC time zone.
```

in integration_tests/src/main/python/marks.py
```
+test_in_premerge_CI_with_non_UTC_TZ = pytest.mark.test_in_premerge_CI_with_non_UTC_TZ
```

Tested:
By default the case with this annotation will run, will not impact existing cases and marks.
`-m this_annotation` will only run cases with this_annotation

Reply:
Removed. Thanks!